pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -8787,3 +8882,107 @@ spec:
 status:
   startTime: %s`, prName, specStatus, now.Format(time.RFC3339)))
 }
+
+func shouldHaveFullEmbeddedStatus(embeddedVal string) bool {
+	return true
+	// TODO(abayer): Uncomment when implementation of TEP-0100 is done.
+	// return embeddedVal == config.FullEmbeddedStatus || embeddedVal == config.BothEmbeddedStatus
+}
+
+func shouldHaveMinimalEmbeddedStatus(embeddedVal string) bool {
+	return false
+	// TODO(abayer): Uncomment when implementation of TEP-0100 is done.
+	// return embeddedVal == config.MinimalEmbeddedStatus || embeddedVal == config.BothEmbeddedStatus
+}
+
+func verifyTaskRunStatusesCount(t *testing.T, embeddedStatus string, prStatus v1beta1.PipelineRunStatus, taskCount int) {
```

Comment:
just one more comment-- t.Helper() would be useful here

Reply:
Good catch!